projects/ad74413r/src/common/common_data.c

```diff
@@ -61,32 +61,4 @@ struct no_os_spi_init_param ad74413r_spi_ip = {
 	.extra = SPI_EXTRA,
 };
 
-#ifdef IIO_TRIGGER_EXAMPLE
```

Comment:
this trigger specific code was moved to the iio_trigger_example code, it should've not been here!